scripts/docker/README

```diff
@@ -0,0 +1,37 @@
+Dockerfiles are used to create Docker images.  Docker images are the 'recipes' used to build virtual environments called containers.
```

Comment:
For readability on github, it might be better to convert this to a markdown file.
(we can take care of this separately).

This is what it looks like now on github: https://github.com/keithhealy/axom/tree/develop/scripts/docker

Reply:
I changed the file to markdown format.

Reply:
Thanks @keithhealy -- the writeup is really helpful.

Could you please rename it to ``README.md`` ?

Reply:
Changed to README.md and fixed some formatting errors.